### PR TITLE
Drone 0.8 and HaaS updates

### PR DESCRIPTION
**Pull request by mdharamadas1**
Fixes: https://github.com/vmware/vic-ui/issues/356

This will be merged only after new drone is active.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,103 +1,79 @@
-# After any change to this file you MUST re-sign and checkin the .drone.yml.sig
```
Comment (jak-atx) [inline .drone.yml]: This is no longer required?

File: .drone.yml [hunk 2]
```diff
@@ -1,103 +1,79 @@
...
+    cache_control: 'public,max-age=3600'
     when:
       repo: vmware/vic-ui
-      event: [ push, tag ]
-      branch: [ refs/tags/* ]
+      event: [push, tag]
+      branch: ['refs/tags/*']
       status: success
 
-  report-coverage:
```
Comment (jak-atx) [inline .drone.yml]: Why is the code coverage section removed?

File: .drone.yml (hunk 1 above)
Comment (mdharamadas1) [reply .drone.yml]: nope

File: .drone.yml (hunk 2 above)
Comment (mdharamadas1) [reply .drone.yml]: That seems like a mistake, I will add it back.

**Pull request merged**